bin/start-history-server.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+set -e
+
+export THS_HOME="$(cd "`dirname "$0"`"/..; pwd)" # root of the project
```

Comment:
Rather than make things _history server_ specific, let's keep everything as just "tony". We should have a single tony configuration file/home that both clients and the history server can share. This is similar to what MapReduce does. A single `HADOOP_MAPRED_HOME` environment variable and the `mapred-site.xml` file contains both client-side configs and job history server configs.

So I think we should name things like:

* TONY_HOME
* tony-env.sh
* tony-site.xml
* etc.